content/id/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -0,0 +1,114 @@
+
+```yaml
+apiVersion: v1
+kind: Pod
+
+...
+spec:
+	schedulerName: poseidon
+``` 
+
+{{< note >}}
+Untuk detail tentang desain proyek ini, lihat [dokumen desain](https://github.com/kubernetes-sigs/poseidon/blob/master/docs/design/README.md).
+{{< /note >}}
+
+## Kemungkinan Skenario Kasus Penggunaan - Kapan menggunakannya
+
+Seperti yang disebutkan sebelumnya, penjadwal Poseidon-Firmament memungkinkan lingkungan penjadwalan dengan _throughput_ yang sangat tinggi pada skala karena pendekatan penjadwalan massal versus pendekatan _pod-at-a-time_ Kubernetes. Dalam pengujian ekstensif kami, kami telah mengamati manfaat _throughput_ substansial selama persyaratan sumber daya (CPU / Memori) untuk _Pod_ yang masuk seragam di seluruh _job_ (_Replicasets_ / _Deployment_ / _Jobs_), terutama karena amortisasi pekerjaan yang efisien di seluruh _job_.
+
+Meskipun penjadwal Poseidon-Firmament mampu menjadwalkan berbagai jenis beban kerja, seperti layanan, batch, dll., berikut ini adalah beberapa kasus penggunaan yang paling unggul:
+
+1. Untuk pekerjaan "Big Data / AI" yang terdiri dari sejumlah besar tugas, manfaat dari _throughput_ luar biasa.
+2. Pekerjaan layanan atau sekumpulan di mana persyaratan sumber dayanya adalah pekerjaan seragam di seluruh pekerjaan (_Replicasets_/_Deployments_/_Jobs_).
+
+## Tahap Proyek Saat Ini
+
+- **Rilis Alpha - Repo Inkubasi.** di https://github.com/kubernetes-sigs/poseidon.  
+- Saat ini, penjadwal Poseidon-Firmament **tidak memberikan dukungan untuk ketersediaan tinggi**, implementasi kami mengasumsikan bahwa penjadwal tidak mungkin gagal. [Dokumen desain](https://github.com/kubernetes-sigs/poseidon/blob/master/docs/design/README.md) menjelaskan cara-cara yang memungkinkan untuk mengaktifkan ketersediaan tinggi, tetapi kami membiarkannya untuk pekerjaan mendatang.
+- Kami **tidak mengetahui adanya production deployment** dari penjadwal Poseidon-Firmament saat ini.
+- Poseidon-Firmament didukung dari rilis Kubernetes 1.6 dan bekerja dengan semua rilis berikutnya.
+- Proses rilis untuk repo Poseidon dan Firmament berada dalam langkah kunci. Rilis Poseidon saat ini dapat ditemukan [di sini](https://github.com/kubernetes-sigs/poseidon/releases) dan rilis Firmament yang sesuai dapat ditemukan [di sini](https://github.com/Huawei-PaaS/firmament/releases).
+
+## Matriks Perbandingan Fitur
+
+| Fitur | Penjadwal Bawaan Kubernetes | Penjadwal Poseidon-Firmament | Catatan |
+|--- |--- |--- |--- |
+|_Node Affinity_/_Anti-Affinity_|Y|Y||
+| _Pod Affinity_ / _Anti-Affinity_ - termasuk dukungan untuk simetri _anti-affinity_ _pod_ | Y | Y | Saat ini penjadwal bawaan mengungguli penjadwal Poseidon-Firmament _pod_ dalam segi fungsionalitas _affinity_/_anti-affinity_. Kami sedang berupaya menyelesaikan ini. |
+|_Taints_ & Toleransi|Y|Y||
```

Comment:
toleration is kubernetes specific, let's not translate this
```suggestion
|_Taint_ & _Toleration_|Y|Y||
```

Reply:
done